float64 xDisp
float64 yDisp
float64 disp
